antlr/src/main/antlr/FSH.g4

```diff
@@ -120,6 +145,12 @@ REFERENCE:          'Reference' WS* '(' WS* SEQUENCE WS* ('|' WS* SEQUENCE WS*)*
                  //  ^  NON-WHITESPACE
 CARET_SEQUENCE:     '^' ~[ \t\r\n\f]+;
 
+                 // '/' EXPRESSION '/'
+REGEX:              '/' ('\\/' | ~[/\r\n])+ '/';
+
+
+COMMA_DELIMITED_CODES: (CODE (WS+ STRING)? WS* COMMA WS+)+ CODE (WS+ STRING)?;
```

Comment:
I see why you needed this, and have no better solution, but it kind of sucks that we're going to have to parse out all these parts in the importer. :-/